docs/serving/accessing-traces.md

```diff
@@ -9,6 +9,8 @@ If you have not yet installed the logging and monitoring components, go through
 the [installation instructions](./installing-logging-metrics-traces.md) to set
```

Comment:
(SEO item to add an accurate description of what is covered in this topic)

Above this conditional statement ("If you have not yet installed..."), can you please add the following (please add the following sentence at row 8 and move everything else down):

Depending on the request tracing tool that you have installed on your Knative Serving cluster, see the corresponding section for details about how to visualize and trace requests.